Command/DumpCommand.php

```diff
@@ -72,7 +78,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $formats = $input->getOption('format');
         $merge = (object) array(
-            'domains' => $input->getOption('merge-domains')
+            'domains' => $input->getOption('merge-domains'),
+            'fallback' => $input->getOption('merge-fallback')
```

Comment:
I suggest adding the trailing comma here, to avoid future diffs like the one done on the previous line